axelrod/graph.py

```diff
@@ -0,0 +1,126 @@
+"""
+Weighted undirected sparse graphs.
+
+Original source:
+https://github.com/marcharper/stationary/blob/master/stationary/utils/graph.py
```

Comment:
Turns out we need the graph to be weighted for some applications in the literature. If the reproduction graph is weighted then it changes the death probabilities in the birth-death case.